baaas-decision-fleet-manager/src/test/java/org/kie/baaas/dfm/app/controller/DecisionResourceRestTest.java

```diff
@@ -156,4 +159,32 @@ void testEventingValidation() {
                 .then()
                 .statusCode(201);
     }
+
+    @TestSecurity(user = DEFAULT_CUSTOMER_ID)
+    @Test
+    void testDecisionRequestNameValidation() {
+        DecisionRequest request = new DecisionRequest();
+        request.setName("1example-request");
+        request.setDescription("the description");
+        request.setKind("Decision");
+        List<String> errors = given()
+                .when()
+                .body(request)
+                .contentType(MediaType.APPLICATION_JSON)
+                .post("/decisions")
+                .then()
+                .statusCode(400)
+                .extract().body().as(ArrayList.class);
+
+        assertThat(errors, hasSize(1));
+
+        request.setName("g-example-request");
+        given()
+                .when()
+                .body(request)
+                .contentType(MediaType.APPLICATION_JSON)
+                .post("/decisions")
+                .then()
+                .statusCode(201);
```

Comment:
Hi @myeung18 , iiuc the decision name of this request is not valid (because the first char is a number) but the status code is `201`. Wouldn't it be better to validate the request as soon as we get it and return a `400`? wdyt? cc @robpblake

Reply:
See my comment above about the types of name I think we should support.

cheers,
Rob